integrations/haproxy/documentation.yaml

```diff
@@ -81,6 +85,9 @@ podmonitoring_config: |
     selector:
       matchLabels:
         app.kubernetes.io/name: haproxy
+additional_podmonitoring_info: |
+  Ensure that the values of the `port` and `matchLabels` fields match those of the {{app_name_short}} pods you want to monitor.
```

Comment:
Not clear if this is one paragraph or two. line 89 and 90 are "thematically" different.

Reply:
I think two paragraphs is good here. Thanks!